### DISCUSSION TITLE
A new design for the right ToC

### DISCUSSION THREAD

**Discussion by janluke** [pradyunsg/furo#167] [Ideas] [answered]
I find the default right ToC a little difficult to read, so I hacked together some CSS to make it look like this: https://cloup.readthedocs.io/en/latest/pages/constraints.html

My current brand color is probably too clear. Don't bother about that. The relevant changes are:
- using a bigger font-size
- increase the indentation of sublists
- increase the vertical space between first-level items
- make the first level pop up using size and color emphasis (it could just be a darker shade of gray).

Don't know what you think but I'm pretty satisfied with the result :)

**Comment by pradyunsg** [comment, marked as answer]
Neat.

I've tweaked the vertical spacing. I'm not a fan of the font-size changes -- the default font size that Furo uses is whatever the user's browser is configured to; and not-the-main-content (like admonitions and right sidebar stuff) gets a smaller size  to reflect that they're less "important". Likewise for the use of colors in the right sidebar.

FWIW, you'll likely need to update the custom CSS you've got there, since beta39 reworked substantial portions of how that worked -- https://pradyunsg.me/furo/changelog/#beta39.

A bunch of your CSS variables should likely live in the configuration file via https://pradyunsg.me/furo/customisation/#light-css-variables-dark-css-variables, since there's no promises on how the various selectors might need to change; but I do make an effort to not break configuration-based workflows. :)

**Reply by janluke** [reply]
I've seen different sources suggesting to increase the default font size for text-heavy websites and I used Google's web.dev as an inspiration for typography and spacing. Thank you for the warnings!
